SU-SWS/stanford_basic
# D8CORE-941: Headings and styles

# READY FOR REVIEW

# Summary
_See ticket:  https://stanfordits.atlassian.net/browse/D8CORE-941_

# Needed Review By (Date)
- Sprint 


# Urgency


# Steps to Test
- Add an image with a caption to the WYSIWYG
- Verify that the caption is centered

- Verify that the styles in 
https://www.figma.com/file/mQ5izx1G6f8DEUkgwWIHWP/D8-End-User-Display?node-id=0%3A1
match what's here, except Drop Cap styles. They're in a separate [PR](https://github.com/SU-SWS/stanford_basic/pull/108).

# Affected Projects or Products
- D8Core (https://github.com/SU-SWS/acsf-cardinalsites)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/D8CORE-941

## Related PRs
Drop cap styles:
https://github.com/SU-SWS/stanford_basic/pull/108
## Accessibility Testing

## More Information
Caveat: I haven't figured out the request at the top. 
Kill the margin-top of any style is first-child and margin-bottom is last-child.
I'll put that in a separate PR.
I'll also accept advice on how to do this as the answer is eluding me.

## Folks to notify
`@sherakama` 
@kerri-augenstein

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
